### PR TITLE
Complete HM1

### PR DESCRIPTION
**Pull request by yuvalorsher**

### CODE REVIEW

**Comment by yuvalorsher** [comment]
PR retry

File: tests/test_hello_world.py [hunk 1]
```diff
@@ -93,17 +95,55 @@ def test_make_me_2_functions_one_use_fixture_and_one_use_parametrize():
         assert times_7(rnd_int) == sum([rnd_int for i in range(7)])
```
Comment (ronwettenstein) [inline tests/test_hello_world.py:95]: Please think about how you take this logic and extract it to a test that uses fixture
(This is the idea of the TODO above)

File: tests/test_hello_world.py [hunk 2]
```diff
@@ -93,17 +95,55 @@ def test_make_me_2_functions_one_use_fixture_and_one_use_parametrize():
...
+    for i in range(1,n+1):
+        f *= i
+    return f
+
+
+@pytest.mark.parametrize("number", [0, 1, 4])
+def test_my_factorial(number):
+    from math import factorial
+    assert my_factorial(number) == factorial(number)
+
```
Comment (ronwettenstein) [inline tests/test_hello_world.py:123]: Add 2 more tests

**Review by ronwettenstein — COMMENTED** [review]
Nice Job :) 
See my comments

**Comment by yuvalorsher** [comment]
@ronwettenstein Addressed your notes

**Pull request closed without merging**